docs/source/training_md.rst

```diff
@@ -0,0 +1,100 @@
+.. _`training_md`:
+
+Machine Description case
+------------------------
+
+In this example we will combine multiple IMAS-ParaView non-GGD plugins to read ITER machine description data. We will create an animation where we orbit the camera around the machine description data, so we can see the different structures clearly from every angle.
+
+You can download the ParaView state file for this example :download:`here <pv_states/md_state.pvsm>`. However, we recommend that you manually follow the steps outlined below.
+
+Loading the Wall data
+^^^^^^^^^^^^^^^^^^^^^
+In this subsection, we will visualize the divertor and first wall structures, and create a rotational extrusion to rotate them around their center axis.
+
+
+.. |ico1| image:: images/rotate_axis.png
+
+#. Navigate to *Sources > IMAS Tools* and select the Wall Limiter Reader.
+#. Enter the following URI in the ``Enter URI`` field of the Wall Limiter reader plugin:
+
+   .. code-block:: bash
+
+      imas:hdf5?path=/work/imas/shared/imasdb/ITER_MACHINE_DESCRIPTION/3/116000/5
+
+#. Since this URI only contains a single supported IDS for this reader, the wall IDS, it is automatically selected for you. You should see the *Divertor* and *First Wall* attributes appear in the attribute array selection, select them both.
+#. Select ``Apply`` to load the *Divertor* and *First Wall* structures.
+#. After the structures are loaded, bring them into view by aligning the viewpoint in the positive Y direction using the following button: |ico1|.
+#. We will now rotate the divertor and first wall around their center axis. For this, we will use the Rotational Extrusion filter. Select the *Filters > Search...* tab, and type ``Rotational Extrusion`` and select the filter.
+#. Set the resolution to 50, and the angle to 360. Press ``Apply`` to apply the rotational extrusion filter.
+#. We will now fix the legend labels for the blocks, select ``Edit`` under the coloring section and remove all the categories for numbers higher than 1, and select the minus icon.
+#. Now rename the annotations for the 0 and 1 blocks to ``Divertor`` and ``First Wall``, respectively.
+#. Lastly, set the opacity to 0.5, so we can visualize the inside of the wall as well.
+ 
+.. figure:: images/training/md_wall_limiter.png
+
+   The first wall and divertor structures extruded around the center axis. Data provided by J. Artola.
```

Comment:
I don't think J provided this data, did he?
Probably each MD IDS has a different data responsible officer at IO.